pkg/sqlite/migrations/006_associate_apis_with_bundle_test.go

```diff
@@ -157,3 +154,41 @@ func oldGetChannelEntriesThatProvide(db *sql.DB, group, version, kind string) (e
 	}
 	return
 }
+
+func newGetChannelEntriesThatProvide(db *sql.DB, group, version, kind string) (entries []*registry.ChannelEntry, err error) {
```

Comment:
this replicates the previous query for the db tests, since the current one uses the properties field (that doesn't exist at this stage of migration)